src/molecules/NumberedList.js

```diff
@@ -9,8 +9,8 @@ const Ol = styled.ol`
 	padding: 0;
 `;
 
-const NumberedList = ({ items, itemProp }) => (
-	<Ol>
+const NumberedList = ({ items, itemProp, className }) => (
+	<Ol className={className}>
```

Comment:
Not sure if anyone on your side needs this for selenium tests. Ask anyone around you in Odessa.

Reply:
It's not for selenium, it's for that thing that we discussed regarding mobile version. Styles for inner elements do not apply unless we do this. 

https://www.styled-components.com/docs/advanced#styling-normal-react-components